components/nds-column-list/scss/column-list.scss

```diff
@@ -0,0 +1,46 @@
+@import '~@nice-digital/nds-core/scss/core';
+
+.column-list {
+  list-style: none;
+  margin: 0;
+  max-width: 100%;
+
+  &--boxed {
+    background: $nds-colour-true-white;
+    border: 1px solid $nds-colour-border;
+
+    a {
+      padding: rem($nds-spacing-x-small $nds-spacing-medium);
+    }
+
+    @include mq($from: md) {
```

Comment:
Feels like these should be BEM too, something like `.column-list--triple` or `.column-list--two-cols` or something as they're a modifier on the root Block, so these duplicated rules can be outside `--plain` and `--boxed`

Reply:
Done!